### DISCUSSION TITLE
How to export GetText Results in a pre-existing Google Sheet

### DISCUSSION THREAD

**Discussion by alpha-code2019** [AutomaApp/automa#531] [Q&A] [answered]
Hello!

1) I wonder how I can populate a pre-existing Google sheet that contains 38 columns with the results of a scraping workflow that has produced values for just 14 of these 38 columns?

2) How can I create a workflow is such a way that the results of each new run of the workflow append to the same Google sheet as in question 1

Thank you for helping to find a solution for these two problems!

**Comment by Kholid060** [comment, marked as answer]
Use the Google sheets block and select the `Append spreadsheet cell values` option
![image](https://user-images.githubusercontent.com/22908993/172074779-5bd5ea46-ec2a-4a9d-8168-be61dba9b48a.png)

**Comment by alpha-code2019** [comment]
Thank you so much!

Am Mo., 6. Juni 2022 um 01:23 Uhr schrieb Ahmad Kholid <
***@***.***>:

**Comment by juliocfc92** [comment]
I want to do something like this, I get the value from column A, search on a website, then it should get the value and fill in column B of the corresponding row, and then loop back to the next row. I'm having problems to set the RANGE. how to now de index? :(

**Reply by juliocfc92** [reply]
![image](https://github.com/user-attachments/assets/4be2f05e-9b47-4efe-8a7a-56342f947d1d)

Consegui.
